twoliter/embedded/Makefile.toml

```
@@ -47,7 +47,7 @@ BUILDSYS_PRETTY_NAME = "Bottlerocket OS"
 # starting now.  (This can be an RFC3339 date, or an offset like "in X
 # hours/days/weeks".)
 PUBLISH_EXPIRATION_POLICY_PATH = "${BUILDSYS_ROOT_DIR}/tools/pubsys/policies/repo-expiration/2w-2w-1w.toml"
-PUBLISH_WAVE_POLICY_PATH = "${BUILDSYS_ROOT_DIR}/sources/updater/waves/default-waves.toml"
+PUBLISH_WAVE_POLICY_PATH = "${BUILDSYS_BUILD_DIR}/tools/default-waves.toml"
```

Can we move these to `tools/waves` instead to keep the tools directly a little less messy?
```suggestion
PUBLISH_WAVE_POLICY_PATH = "${BUILDSYS_BUILD_DIR}/tools/waves/default-waves.toml"
```

---

Let me try and see

---

Done, testing assumptions now